From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:52 +0100
Subject: python-XenAPI-install-path

---
 scripts/examples/python/OMakefile |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

--- a/scripts/OMakefile
+++ b/scripts/OMakefile
@@ -1,3 +1,4 @@
+#foo
 SITE_DIR=$(shell python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
 
 .PHONY: noarch-install install sdk-install
